torchvision/prototype/transforms/functional/_color.py

```diff
@@ -32,6 +40,10 @@ def adjust_saturation(inpt: features.InputTypeJIT, saturation_factor: float) ->
 adjust_contrast_image_pil = _FP.adjust_contrast
 
 
+def adjust_contrast_video(video: torch.Tensor, contrast_factor: float) -> torch.Tensor:
+    return adjust_contrast_image_tensor(video, contrast_factor=contrast_factor)
```

Comment:
Question: Should we alias it instead? On the pros side, if someone asks for the name of the method, they will get `adjust_contrast_video` which is what we want. On the cons side, we just have an extra unnecessary method call. No strong opinion, just highlighting it.

Reply:
>  we just have an extra unnecessary method call. 

That will hardly make a difference. I think we can afford an extra kernel for now. In the future we can revisit this. Especially, if we support 2d images everywhere, most mask kernels could also be aliased. The only exceptions that come to mind are the ones with an interpolation mode.